### DISCUSSION TITLE
Add /playtime(resolved)

### DISCUSSION THREAD

**Discussion by RealHypnoticOcelot** [EssentialsX/Essentials#4473] [Ideas and Feature Suggestions] [answered]
Add /playtime including a players first join date, playtime, and last join, available with essentials.playtime

**Comment by RealHypnoticOcelot** [comment]
yes but i don’t want players to access /whois i want them to see the playtime and the playtime only

**Comment by ghost** [comment, marked as answer]
nevermind just used a third party playtime plugin
